components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -115,8 +123,10 @@ int RRTMGPRadiation::requested_buffer_size_in_bytes() const
                                 Buffer::num_2d_nlay*m_ncol*m_nlay*sizeof(Real) +
                                 Buffer::num_2d_nlay_p1*m_ncol*(m_nlay+1)*sizeof(Real) +
                                 Buffer::num_2d_nswbands*m_ncol*m_nswbands*sizeof(Real) +
-                                Buffer::num_3d_nswbands*m_ncol*(m_nlay+1)*m_nswbands*sizeof(Real) +
-                                Buffer::num_3d_nlwbands*m_ncol*(m_nlay+1)*m_nlwbands*sizeof(Real);
+                                Buffer::num_3d_nlev_nswbands*m_ncol*(m_nlay+1)*m_nswbands*sizeof(Real) +
```

Comment:
I don't see where there is an (m_nlay+1) sized array being added to the buffer.  I only see the four buffer arrays of size `m_ncol*m_nlay*m_ns/lwbands`.  Am I missing something?

Reply:
The `num_3d_nlev_nswbands` fields (`m_nlay+1`) are the existing 3d fluxes that were already in the buffer, the `num_3d_nlay_nswbands` fields (`m_nlay`) are the new ones that hold the aerosol optical properties that were added in this PR. I changed the name for the existing ones from `num_3d_nswbands` to `num_3d_nlev_nswbands` to distinguish between the number with `m_nlay` and `m_nlay+1` vertical levels. Does this make sense, or am I still missing something?

Reply:
ahhh wait I see.  I didn't catch the nuance between the before and after.  I thought this was a whole new line added and then looked at the new variables being added below and couldn't figure out where the memory was used.  But now I see, these were already there and this isn't a new line, just a change of name.  Thanks for clarifying.